packages/office-addin-dev-settings/src/appcontainer.ts

```diff
@@ -120,13 +122,30 @@ export function getDisplayNameFromManifestPath(manifestPath: string): string {
 }
 
 export async function ensureLoopbackIsEnabled(manifestPath: string, askForConfirmation: boolean = true): Promise<boolean> {
-  const name = await getAppcontainerNameFromManifestPath(manifestPath);
-  let isEnabled = await isLoopbackExemptionForAppcontainer(name);
-
-  if (!isEnabled) {
-    if (!askForConfirmation || await getUserConfirmation(getDisplayNameFromManifestPath(manifestPath))) {
-      await addLoopbackExemptionForAppcontainer(name);
-      isEnabled = true;
+  let isEnabled = false;
+  try {
+    const name = await getAppcontainerNameFromManifestPath(manifestPath);
+    isEnabled = await isLoopbackExemptionForAppcontainer(name);
+  
+    if (!isEnabled) {
+      if (!askForConfirmation || await getUserConfirmation(getDisplayNameFromManifestPath(manifestPath))) {
+        await addLoopbackExemptionForAppcontainer(name);
+        isEnabled = true;
+      }
+    }
+    // Until changes to usage-data get approved, this convention will do
+    // usageDataObject.sendUsageDataSuccessEvent("ensureLoopbackIsEnabled")
+    usageDataObject.sendUsageDataSuccessEvent("ensureLoopbackIsEnabled", {Pass: true})
+  } catch (error) {
+    if (error == new Error(`The source location could not be retrieved from the manifest.`)
+    || error == new Error(`Please provide the path to the manifest file.`)
+    || (error.message && (<string>(error.message)).startsWith("Platform not supported:"))) {
+      // Until changes to usage-data get approved, this convention will do
```

Comment:
Regarding this command, is there another PR out to add sendUsageDataSuccessfulFailEvent?

Reply:
I'll send one shortly

Reply:
https://github.com/OfficeDev/Office-Addin-Scripts/pull/366